UnityEngine.AssetBundles/Editor/AssetBundleBrowser/InspectTab/AssetBundleInspectTab.cs

```diff
@@ -161,12 +200,20 @@ private void ClearData()
         {
             m_SingleInspector.SetBundle(null);
 
-            foreach (var bundle in m_LoadedBundles)
+            if (null != this.m_loadedAssetBundles)
             {
-                if (bundle != null) //get into this situation on a rare restart weirdness.
-                    bundle.Unload(true);
+                List<string> paths = new List<string>(m_loadedAssetBundles.Keys);
```

Comment:
This would be more efficient as a:
```
List<AssetBundle> bundles = new List<AssetBundle>(m_loadedAssetBundles.Values);
```
then you just loop it and call unload on each.  In your current implementation you have to do a lookup in the dictionary on every round of the loop. Minor, but seems worth doing the other way.